pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -56,6 +56,8 @@ while (( "$n" < "$nParams" )); do
                 c++*) isCxx=1 ;;
             esac
             ;;
+        # check common c++ file extensions
+        *.C|*.H|*.cc|*.CC|*.cp|*.hh|*.ccm|*.cpp|*.CPP|*.c++|*.C++|*.cxx|*.CXX|*.hpp|*.hxx|*.c++m|*.cppm|*.cxxm) isCxx=1 ;;
```

Comment:
I don't think that `isCxx` is correct from looking at what macOS clang does. it seems like we just want to bring in the c++ include file if these file extensions are set -- also where did the list of file extensions come from? they should match the ones clang uses.

Reply:
Hey @paparodeo, those are taken from what [clang](https://github.com/llvm/llvm-project/blob/main/clang/lib/Driver/Types.cpp#L299-L369) checks. I could add a comment for more context.

Reply:
> Edit: What should be done instead of `isCxx`? From what I've seen, it should behave like `-xc++` which is what clang does.

you're right. I thought `isCxx` was adding `-lc++` but it isn't.